server/sonar-web/src/main/js/components/common/MultiSelect.tsx

```diff
@@ -247,20 +246,15 @@ export default class MultiSelect extends React.PureComponent<Props, State> {
   };
 
   render() {
-    const {
-      allowSelection = true,
-      allowNewElements = true,
-      displayFooterNode = false,
-      footerNode = ''
-    } = this.props;
+    const { allowSelection = true, allowNewElements = true, footerNode = '' } = this.props;
     const { query, activeIdx, selectedElements, unselectedElements } = this.state;
     const activeElement = this.getAllElements(this.props, this.state)[activeIdx];
     const infiniteList = this.props.listSize === 0;
     const listClasses = classNames('menu', {
       'menu-vertically-limited': infiniteList,
       'spacer-top': infiniteList,
       'with-top-separator': infiniteList,
-      'with-bottom-separator': displayFooterNode
+      'with-bottom-separator': footerNode !== ''
```

Comment:
`footerNode` is a `React.ReactNode`, so it could be `undefined`. I'd check with `Boolean(footerNode)` : so if it's `undefinded` or `''` it will be false.